pkg/kv/kvserver/replica_test.go

```diff
@@ -6994,7 +6994,7 @@ func TestReplicaLoadSystemConfigSpanIntent(t *testing.T) {
 		if found {
 			return nil
 		}
-		return errors.New("recent write not found in gossiped SystemConfig")
+		return errors.New("recent write not found in gossiped system.descriptor and system.zones spans")
```

Comment:
See below, I don't think we need to gossip these writes anymore so this test could be trimmed down.